license_manager/proto/license_manager.proto

```diff
@@ -175,6 +192,11 @@ message LicenseStats {
   // Number of licenses currently allocated, as reported by the
   // LicenseManager. This should be less than or equal to in_use_count.
   uint32 allocated_count = 5;
+
+  // Number of invocations queued for a license, as reported by the
+  // LicenseManager. If this number is >0, then allocated_count should equal
+  // total_license_count.
+  uint32 queued_count = 6;
```

Comment:
I would consider returning the queue itself, <invocation id, owner, build tag> ... so the tool can print the queue itself. If this queue ever grows to more than O(10)s of items, we are in trouble, so I suspect returning the array to be just fine?

Reply:
I realized the same when I went to implement it - didn't want to change the proto any more than necessary to implement what was already approved. Future PR ok?

Reply:
SGTM